Dockerfile

```diff
@@ -0,0 +1,107 @@
+
+USER root
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    TERM=linux \
+    INITRD=No \
+    LANGUAGE=en_US.UTF-8 \
+    LANG=en_US.UTF-8  \
+    LC_ALL=en_US.UTF-8 \
+    LC_CTYPE=en_US.UTF-8 \
+    LC_MESSAGES=en_US.UTF-8 \
+    LC_ALL=en_US.UTF-8 \
+    GOSU_VERSION=1.9 \
+    TINI_VERSION=v0.13.2 \
+    TOX_VERSION=3.2.1
+
+#add root asset (aliases and fix user id)
+ADD files/* /root/
+
+
+# Install curl, locales, apt-utils, gosu and tini
+# create en_US.UTF-8
+# update distribution package
+# add few common alias to root user
+# add utilities (create user, post install script)
+# create airdock user list
+RUN set -x && \
+  apt-get update -qq && \
+  apt-get install -y \
+    apt-utils \
+    curl \
+    locales \
+    build-essential \
+    git 
+
+# The mkdir statement is necessary, more info here:
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
+RUN set -x && \
+  mkdir -p /usr/share/man/man1 && \ 
+  apt-get update -qq && \
+  apt-get install -y \
+  default-jdk
+
+RUN set -x && \
+  apt-get install -y --no-install-recommends ca-certificates wget vim && \
+  sed -i 's/^# en_US.UTF-8 UTF-8$/en_US.UTF-8 UTF-8/g' /etc/locale.gen && locale-gen && \
+  update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 && \
+  apt-get update -y && \
+  dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')"  && \
+  wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch"  && \
+  wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc"  && \
+  export GNUPGHOME="$(mktemp -d)"  && \
+  echo "disable-ipv6" >> "${GNUPGHOME}/dirmngr.conf" && \
+  gpg --no-tty --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4  && \
+  gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu  && \
+  rm -r /usr/local/bin/gosu.asc  && \
+  chmod +x /usr/local/bin/gosu  && \
+  gosu nobody true  && \
+  wget -O /usr/local/bin/tini "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini" && \
+  wget -O /usr/local/bin/tini.asc "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc" && \
+  gpg --no-tty --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 && \
+  gpg --batch --verify /usr/local/bin/tini.asc /usr/local/bin/tini  && \
+  rm -r "$GNUPGHOME" /usr/local/bin/tini.asc  && \
+  chmod +x /usr/local/bin/tini  && \
+  apt-get purge -y --auto-remove wget  && \
+  mv /root/aliases /root/.aliases && \
+  echo "source ~/.aliases" >> /root/.bashrc && \
+  /root/create-user redis 4201 redis 4201  && \
+  /root/create-user elasticsearch 4202 elasticsearch 4202 && \
+  /root/create-user mongodb 4203 mongodb 4203 && \
+  /root/create-user rabbitmq 4204 rabbitmq 4204 && \
+  /root/create-user java 4205 java 4205 && \
+  /root/create-user py 4206 py 4206 && \
+  /root/create-user node 4207 node 4207 && \
```

Comment:
Indeed it is for this scope. I wasn't sure if we had any internal dependencies somewhere. For the sake of this image, anything unrelated should be removed. 

I think `redis` and `node` should be removed as well. Not because they aren't useful but because it's out of scope of this image. 

WDYT ?

Reply:
I didn't want to ask to remove everything, but yes: only keep what we need for now, it'll be easy to add them back if we ever need it in the future